subsys/bluetooth/mesh/transport.c

```diff
@@ -982,6 +982,10 @@ static int trans_unseg(struct net_buf_simple *buf, struct bt_mesh_net_rx *rx,
 		return -EINVAL;
 	}
 
+	if (!rx->local_match) {
```

Comment:
Could this check be removed?
https://github.com/zephyrproject-rtos/zephyr/blob/cd3c3cf6ebc136e8672220382a0c59622b51531b/subsys/bluetooth/mesh/transport.c#L756-L760

Reply:
Thanks

Reply:
Move to this more suitable.